packages/grid/scss/_mixins.scss

```diff
@@ -159,7 +159,7 @@
 // No gutter
 // -----------------------------------------------------------------------------
 
-/// Add `no-gutter` and `no-gutter--{left,right}` classes to the output CSS. These
+/// Add `no-gutter` and `no-gutter--{start,end}` classes to the output CSS. These
```

Comment:
I think we'll need to keep the previous classes as well otherwise it'll be a breaking change, maybe with a note to remove in v11?